{
    "docs": "https://docs.ansible.com/ansible-core/2.16/dev_guide/testing/sanity/pep8.html", 
    "results": [
        {
            "message": "The test `ansible-test sanity --test pep8` [[explain](https://docs.ansible.com/ansible-core/2.16/dev_guide/testing/sanity/pep8.html)] failed with 12 errors:", 
            "output": "plugins/modules/hello_module.py:36:1: E302: expected 2 blank lines, found 1\nplugins/modules/hello_module.py:53:1: E305: expected 2 blank lines after class or function definition, found 1\nplugins/modules/hello_module.py:55:1: W391: blank line at end of file\ntests/unit/modules/test_hello_module.py:6:1: E265: block comment should start with '# '\ntests/unit/modules/test_hello_module.py:15:1: E302: expected 2 blank lines, found 1\ntests/unit/modules/test_hello_module.py:20:1: E302: expected 2 blank lines, found 1\ntests/unit/modules/test_hello_module.py:23:1: E302: expected 2 blank lines, found 1\ntests/unit/modules/test_hello_module.py:26:1: E302: expected 2 blank lines, found 1\ntests/unit/modules/test_hello_module.py:30:1: W293: blank line contains whitespace\ntests/unit/modules/test_hello_module.py:34:1: W293: blank line contains whitespace\ntests/unit/modules/test_hello_module.py:38:1: W293: blank line contains whitespace\ntests/unit/modules/test_hello_module.py:43:1: W391: blank line at end of file"
        }
    ], 
    "verified": false
}
